features/project/project_controller.rb

```diff
@@ -29,9 +31,11 @@ class ProjectController < AuthenticatedControllerBase
       build_runner = FastlaneBuildRunner.new(
         project: project,
         sha: current_sha,
-        github_service: current_github_provider_credential
+        github_service: FastlaneCI::GitHubService.new(provider_credential: current_github_provider_credential)
```

Comment:
Hah.

Reply:
types, am I right?

Reply:
I'm gonna build a typechecker for ruby. I'll call it `JavaCop`